src/Orleans.Core/DistributedTracing/EventSourceEvents.cs

```diff
@@ -1,71 +1,57 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
 using System.Text;
 
-namespace Orleans.DistributedTracing.EventSourceEvents
+namespace Orleans.Runtime
 {
-    [EventSource(Name = "Microsoft-Orleans-CallBackDataEvent")]
-    public class OrleansCallBackDataEvent : EventSource
+    [EventSource(Name = "Microsoft-Orleans-CallBackData")]
+    internal sealed class OrleansCallBackDataEvent : EventSource
     {
         public static readonly OrleansCallBackDataEvent Log = new OrleansCallBackDataEvent();
-        public void OnTimeoutStart()
+        public void OnTimeout()
         {
             WriteEvent(1);
         }
-        public void OnTimeoutStop()
+
+        public void OnTargetSiloFail()
         {
             WriteEvent(2);
         }
 
-        public void OnTargetSiloFailStart()
+        public void DoCallback()
```

Comment:
Is it right to call this "OnSuccess" or not really?

(we can rename later)

Reply:
I think we should make this as simple as possible, this is a 1 to 1 map to the method name it is used in. I'm of the opinion that event source should be like logging, let's not complicate this